provision/initramfs/capabilities/setup-filesystems/20-filesystems

```diff
@@ -220,6 +258,20 @@ update_fstab_and_mount() {
       IFS=","
     ;;
 
+    raid0*|raid1*|raid4*|raid5*|raid6*)
+      # Run any accumulated parted commands before continuing with mkfs
```

Comment:
Nit, update comment to "...before continuing with raid" or similar

Reply:
ok